tests/ui/consts/const-float-bits-conv.rs

```diff
@@ -4,7 +4,8 @@
 #![feature(const_float_bits_conv)]
 #![feature(const_float_classify)]
 #![allow(unused_macro_rules)]
-
+#![feature(f16)]
+#![feature(f128)]
```

Comment:
Nit: put these `#![feature(...)]` gates directly under the existing `#![feature(...)]` gates at the top, rather than after the `#![allow(...)]`